applications/feedback_sentiment_analyzer/cdk/lib/stack.ts

```diff
@@ -9,100 +9,115 @@ import { AppCloudFrontWebsite } from "./constructs/app-cloud-front-website";
 import { AppEnvLambda } from "./constructs/app-env-lambda";
 import {
   AccessLogFormat,
+  CognitoUserPoolsAuthorizer,
   Cors,
   JsonSchemaType,
   JsonSchemaVersion,
   LogGroupLogDestination,
   Model,
   RestApi,
 } from "aws-cdk-lib/aws-apigateway";
-import { Distribution } from "aws-cdk-lib/aws-cloudfront";
-import { RestApiOrigin } from "aws-cdk-lib/aws-cloudfront-origins";
+import {
+  CachePolicy,
+  Distribution,
+  OriginRequestPolicy,
+  ViewerProtocolPolicy,
+} from "aws-cdk-lib/aws-cloudfront";
+import { RestApiOrigin, S3Origin } from "aws-cdk-lib/aws-cloudfront-origins";
 import { LogGroup } from "aws-cdk-lib/aws-logs";
 import { AppAuth } from "./constructs/app-auth";
 import { AppRoutes } from "./constructs/app-routes";
-import { Empty } from "./constructs/app-api-models";
+import { Empty, EnvModel } from "./constructs/app-api-models";
 
 export class AppStack extends Stack {
   constructor(scope: Construct) {
     const prefix = `fsa-${PREFIX}`;
     super(scope, prefix);
 
+    // Create AWS Lambda functions.
+    const fnConfigs = getFunctionConfigs(APP_LANG);
+    const appLambdas = new AppLambdas(this, "fn", fnConfigs);
+
+    // Create state machine.
+    new AppStateMachine(this, prefix, appLambdas.functions);
+
     // Create API
-    const logGroup = new LogGroup(this, `api-log-group`);
+    // const logGroup = new LogGroup(this, `api-log-group`);
     const api = new RestApi(this, `${prefix}-api`, {
       defaultCorsPreflightOptions: {
         allowOrigins: Cors.ALL_ORIGINS,
         allowCredentials: true,
       },
       deployOptions: {
-        accessLogDestination: new LogGroupLogDestination(logGroup),
-        accessLogFormat: AccessLogFormat.jsonWithStandardFields(),
+        // accessLogDestination: new LogGroupLogDestination(logGroup),
```

Comment:
Why comment these out?

Reply:
These require an account level setting "cloudwatch log role" that I didn't have turned on and didn't get around to setting. We need to document it if we include them in the end product.